tests/test_flow/conftest.py

```diff
@@ -15,17 +15,28 @@
 
 
 # Parameterizing a fixture adds the parameter in the test name at the end,
-# like test_name[PARALLEL] and test_name[SERIAL]. This is super helpful while
-# debugging and much clearer than parameterizing `parallel_execution_enabled`
-# which suffixes [TRUE] / [FALSE].
+# like test_name[PARALLEL], test_name[SERIAL], and test_name[AIP]. This is
+# super helpful while debugging and much clearer than parameterizing
+# `parallel_execution_enabled` which suffixes [TRUE] / [FALSE].
 @pytest.fixture(
     params=[
         pytest.param("serial", marks=pytest.mark.serial),
         pytest.param("parallel", marks=pytest.mark.parallel),
+        pytest.param("aip", marks=pytest.mark.aip),
```

Comment:
What's the purpose of doing this with parametrization if every AIP test is explicitly marked as such? It's not clear to me why we're doing this differently from how we do it with GCS access.